examples/demo13.py

```diff
@@ -0,0 +1,378 @@
+        # have the entries add to unity and hence must be normalized inside the routine.
+
+        final_image.applyIPC(IPC_kernel=wfirst.ipc_kernel,edge_treatment='extend',
+                             kernel_normalization=True)
+        # Here, we use `edge_treatment='extend'`, which pads the image with zeros before
+        # applying the kernel. The central part of the image is retained.
+        logger.debug('Applied interpixel capacitance to {0}-band image'.format(filter_name))
+        final_image_5 = final_image.copy()
+        # Isolate the changes due to the interpixel capacitance effect.
+        diff = final_image_5-final_image_4
+
+        out_filename = os.path.join(outpath,'demo13_IPC_{0}.fits'.format(filter_name))
+        final_image_5.write(out_filename)
+        out_filename = os.path.join(outpath,'demo13_diff_IPC_{0}.fits'.format(filter_name))
+        diff.write(out_filename)
+
+        # 5) Adding read noise:
+        # Read noise is the noise due to the on-chip amplifier that converts the charge into an
+        # analog voltage.
+        read_noise = galsim.CCDNoise(rng)
+        read_noise.setReadNoise(wfirst.read_noise)
+        final_image.addNoise(read_noise)
+        final_image_6 = final_image.copy()
+
+        logger.debug('Added readnoise to {0}-band image'.format(filter_name))
+
+        # Technically we have to apply the gain, dividing the signal in e- by the voltage gain
+        # in e-/ADU to get a signal in ADU.  For WFIRST the gain is expected to be around 1,
+        # so it doesn't really matter, but for completeness we include this step.
+        final_image /= wfirst.gain
+
+        # Finally, the analog-to-digital converter reads in an integer value.
+        final_image.quantize()
+        # Note that the image type after this step is still a float.  If we want to actually
+        # get integer values, we can do new_img = galsim.Image(final_image, dtype=int)
+
+        # Since many people are used to viewing background-subtracted images, we provide a
+        # version with the background subtracted (also rounding that to an int).
+        tot_sky_level = (sky_level_pix + wfirst.dark_current*wfirst.exptime)/wfirst.gain
+        tot_sky_level = numpy.round(tot_sky_level)
+        final_image -= tot_sky_level
+
+        logger.debug('Subtracted background for {0}-band image'.format(filter_name))
+        # Write the final image to a file.
+        out_filename = os.path.join(outpath,'demo13_{0}.fits'.format(filter_name))
+        final_image.write(out_filename)
+
+        logger.info('Completed {0}-band image.'.format(filter_name))
+
+    logger.info('You can display the output in ds9 with a command line that looks something like:')
+    logger.info('ds9 -rgb -blue -scale limits -0.2 0.8 output/demo13_J129.fits -green '
+        +'-scale limits'+' -0.25 1.0 output/demo13_W149.fits -red -scale limits -0.25'
+        +' 1.0 output/demo13_Z087.fits -zoom 2 &')
```

Comment:
@arunkannawadi - can you please look at this comment from Mike?
My version of ds9 is so old that it won't take the -rgb command-line option, so I had been ignoring these output statements entirely.  It looks to me like he is correct but since you've actually been using these, please check and fix.  (Make sure to pull the latest version, there are many code updates.)